app/tests/test_items_endpoint.py

```diff
@@ -682,6 +682,44 @@ def test_item_endpoint_put_update_to_datetime_range(self):
         self.assertNotIn('datetime', json_data['properties'].keys())
         self.assertNotIn('title', json_data['properties'].keys())
 
+    def test_item_endpoint_put_remove_properties_title(self):
+        path = f'/{STAC_BASE_V}/collections/{self.collection["name"]}/items/{self.item["name"]}'
+        sample = self.factory.create_item_sample(
+            self.collection.model,
+            sample='item-2',
+            name=self.item['name'],
+            properties={
+                "title": None,
+                "start_datetime": "2020-10-18T00:00:00Z",
+                "end_datetime": "2020-10-19T00:00:00Z",
+            }
+        )
+        response = self.client.put(
+            path, data=sample.get_json('put'), content_type="application/json"
+        )
+        json_data = response.json()
+        self.assertStatusCode(200, response)
+        self.assertEqual(self.item['name'], json_data['id'])
+        self.assertNotIn("title", json_data['properties'].keys())
+
+        sample = self.factory.create_item_sample(
+            self.collection.model,
+            sample='item-2',
+            name=self.item['name'],
+            properties={
+                "title": "item title",
+                "start_datetime": "2020-10-18T00:00:00Z",
+                "end_datetime": "2020-10-19T00:00:00Z",
+            }
+        )
+        response = self.client.put(
+            path, data=sample.get_json('put'), content_type="application/json"
+        )
+        json_data = response.json()
+        self.assertStatusCode(200, response)
+        self.assertEqual(self.item['name'], json_data['id'])
+        self.assertIn("title", json_data['properties'].keys())
```

Comment:
Should it not be the other way around
1. Create an item with title
2. Update the item with PUT `title: null`

Reply:
Or probably both?
Check, that with PUT the title of an existing item with title can be "nulled" and also check, that with PUT an item with title=None can be created from scratch.